README.md

```diff
@@ -40,26 +40,28 @@ class Robot:
     def __init__(self):
 
         self._env = HardwareEnvironment()
-        
+
         self.power_board = PowerBoard(self._env)
-        
+
         self.motor_boards = BoardGroup(MotorBoard, self._env)
         self.motor_board = self.motor_boards.singular()
-        
+
         self.servo_boards = BoardGroup(ServoBoard, self._env)
         self.servo_board = self.servo_boards.singular()
-        
+
         self.ruggeduino = Ruggeduino(self._env)
 
 ```
 
 ## Competitions
 
-We are working with developers for the following robotics competition to support their hardware(1):
```

Comment:
Pretty sure I'm a developer from SR? and @mildlyincompetent

Reply:
Are any of us from RoboCon? I feel that Jake's version is somewhat clearer.

Reply:
That's true, none of are from RoboCon currently. Except Joe but he isn't really involved with `j5`

Reply:
I'm happy for this change to go ahead to be clear.